File: app/src/main/java/mb00/android/codehub/ui/user/inject/UserOverviewFragmentComponent.java
package mb00.android.codehub.ui.user.inject;

import dagger.Subcomponent;
import dagger.android.AndroidInjector;
import mb00.android.codehub.ui.user.view.UserOverviewFragment;


@Subcomponent(modules = UserOverviewFragmentModule.class)
public interface UserOverviewFragmentComponent extends AndroidInjector<UserOverviewFragment> {

    @Subcomponent.Builder
    abstract class Builder extends AndroidInjector.Builder<UserOverviewFragment> {}

}

File: app/src/main/java/mb00/android/codehub/ui/user/inject/UserOverviewFragmentComponent.kt
package mb00.android.codehub.ui.user.inject

import dagger.Subcomponent
import dagger.android.AndroidInjector
import mb00.android.codehub.ui.user.view.UserOverviewFragment


@Subcomponent(modules = [
    UserOverviewFragmentModule::class
])
interface UserOverviewFragmentComponent : AndroidInjector<UserOverviewFragment> {

    @Subcomponent.Builder
    abstract class Builder : AndroidInjector.Builder<UserOverviewFragment>()

}
